include/states.h

```diff
@@ -14,4 +14,16 @@ enum COMPILER_PHASE {
     CP_TOTAL_PHASES,
 };
 
+enum DFA_STATES {
+	DS_ERROR = 0,
+	DS_START,
+	DS_ACCEPT
```

Comment:
It's good practice to put a comma after the last value in the enumeration, so that if we add another value later, it keeps the diff clean

tl;dr; - add a comma after DS_ACCEPT